features/importers/venepaikka_harbors/importer.py

```diff
@@ -0,0 +1,215 @@
+from typing import Iterable, Mapping
+
+import jmespath
+import requests
+from django.conf import settings
+from django.contrib.gis.geos import Point
+from django.utils import timezone
+
+from categories.models import Category
+from features.enums import FeatureTagSource
+from features.importers.base import FeatureImporterBase
+from features.importers.venepaikka_harbors import app_settings
+from features.models import ContactInfo, Feature, Image, License, SourceType, Tag
+
+query = """
+query Harbors {
+  harbors {
+    edges {
+      node {
+        id
+        geometry {
+          type
+          coordinates
+        }
+        properties {
+          name
+          wwwUrl
+          imageLink
+          imageFile
+          streetAddress
+          zipCode
+          municipality
+          phone
+          email
+          servicemapId
+        }
+      }
+    }
+  }
+}
+"""
+
+feature_expression = jmespath.compile(
+    """
+data.harbors.edges[*].node.{
+    id: id,
+    name: properties.name,
+    url: properties.wwwUrl,
+    lat: geometry.coordinates[1],
+    lon: geometry.coordinates[0],
+    address: {
+        street_address: properties.streetAddress,
+        postal_code: properties.zipCode,
+        municipality: properties.municipality
+        phone_number: properties.phone
+        email: properties.email
+    },
+    images: [properties.imageFile, properties.imageLink]
+    servicemap_id: properties.servicemapId
+}
+"""
+)
+
+
+class VenepaikkaImporter(FeatureImporterBase):
+
+    source_system = "venepaikka"
+    source_type = "harbor"
+
+    servicemap_url = "https://palvelukartta.hel.fi/fi/unit/"
+    servicemap_link_type = "servicemap"
+    image_copyright_owner = "venepaikat.hel.fi"
+
+    def import_features(self):
+        source_type = self.get_source_type()
+        client = VenepaikkaHarborsClient()
+        harbors = client.fetch_harbors(query=query).json()
+
+        for harbor in feature_expression.search(harbors):
+            feature = self._import_feature(harbor, source_type)
+            self._import_feature_category(feature, app_settings.CATEGORY_CONFIG)
+            self._import_feature_tag(feature, app_settings.TAG_CONFIG)
```

Comment:
if all of the features get the same tag and category, wouldn't it be DRYer to create `Category` and `Tag` objects before this loop and then assign them to all the features?